tldw_Server_API/app/api/v1/endpoints/data_tables.py

```
@@ -0,0 +1,1126 @@
+@router.get(
+    "",
+    response_model=DataTablesListResponse,
+    summary="List data tables",
+    dependencies=[Depends(require_permissions(MEDIA_READ))],
+)
+async def list_data_tables(
+    status_filter: Optional[str] = Query(None, description="Filter by status"),
+    search: Optional[str] = Query(None, description="Search by name/description"),
+    limit: int = Query(50, ge=1, le=500),
+    offset: int = Query(0, ge=0),
+    current_user: User = Depends(get_request_user),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)